Server/Plugins/APIDump/InfoFile.html

```diff
@@ -217,6 +225,24 @@
 
 
 			<hr />
+			<a name="Categories"><h2>Categories</h2></a>
+
+			<p>The optional Categories table provides descriptions for categories in the generated documentation. Commands can have categories with or without category descriptions in this table. The documentation generator will output a table of listed categories along with their description.</p>
```

Comment:
I completely agree. I've made a few more commits, one of which uses your paragraph.